TLM/TLM/Resources/incompatible_mods.txt

```diff
@@ -79,4 +79,5 @@
 2730687809;Network Extensions 3 - contains malware targeting TM:PE devs
 2399343344;Harmony (redesigned) - by same author of the malware, and potentially much more dangerous
 2389228470;Transfer Broker BETA - by same author of the malware
-2325122732;Supply Chain Coloring - by same author of the malware
\ No newline at end of file
+2325122732;Supply Chain Coloring - by same author of the malware
```

Comment:
Will TM:PE compat checker print this message? Put full message in each line
Imagine it will be printing "by same author" without context?

Reply:
No, I think it pulls the mod name from the mod; those comments in the file are just to make it easier for us to maintain the list of workshop ids. I'll check though just to be sure.

Reply:
Ok, checked and the displayed text is always based on the plugin info / mod name (the text in the .txt file is never shown to user): 
![image](https://user-images.githubusercontent.com/1386719/153517228-1ad77002-c796-44bc-a15b-24ff822637f5.png)

Reply:
I'd still be rather professional about it ;D

Reply:
It's not out of spite; if user has two TM:PE mods subbed at same time, even if one is disabled, it can cause huge problems with save/load of mod otions and road customisations due to duplicate assembly bug (Mono loads both TMPEs due to different build number in the assembly info).

Reply:
I think we can remove the two older chaos mods as they are locked from updates and don't seem to contain anything untoward (that I can find anyway).

I'll send additional commit to remove them from list.

Reply:
I'm just talking about the "- by same author of the malware".
Incompatible mods should of course be marked, no matter what.
But it would be more helpful to write "- due to Patching method xyz" or "- due to broken functionality reported by x" or something like that.